### DISCUSSION TITLE
Lab Assignment #4 👩‍💻

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#51] [Labs]
## Due 7/24  before class 

Please use this thread to post your [lab assignment's #4](https://albertkun.github.io/24SU-ASIAAM-191A/assignments/week4/lab_assignment/) link, which is the GitHub pages URL. [Click here for a reminder on how to get it](https://albertkun.github.io/24SU-ASIAAM-191A/labs/week1/4/#using-github-pages).

### For example

https://albertkun.github.io/24su-asiaam-191a-sample/week4/index.html

**Comment by kimada62** [comment]
https://kimada62.github.io/24su-asiaam-191a-labs/week4/

**Comment by stinest** [comment]
https://stinest.github.io/24su-191a-labs/week4/index.html

**Comment by sjeyakum** [comment]
https://sjeyakum.github.io/24su-aas191a-labs/week4/index.html

**Comment by Keiya-Wada** [comment]
https://keiya-wada.github.io/asiaam-labs/week4/index.html

**Comment by pwinnie2026** [comment]
https://pwinnie2026.github.io/24su-asiaam-191a-labs/Week4Extra/index.html

**Comment by giaari15** [comment]
https://giaari15.github.io/aas191-labs/week_4/week_3/index.html

**Comment by nesslee123** [comment]
https://nesslee123.github.io/Assignments/week4/index.html

**Comment by zhamic7** [comment]
https://zhamic7.github.io/aa191a-labs/week4/index.html

**Comment by kianaptran** [comment]
https://kianaptran.github.io/24su-asiaam-191a-labs/week4/

**Comment by jasminereyz** [comment]
https://jasminereyz.github.io/AA191-Labs/week4/

**Comment by chrissanrow** [comment]
https://chrissanrow.github.io/asiaam191a-su24/week4/

**Comment by kchhoeu** [comment]
https://kchhoeu.github.io/24su-asiaam191a-labs/week4/

**Comment by NellieNguyen149** [comment]
https://nellienguyen149.github.io/24su-asianam-191a-labs/week4/index.html

sorry... ;-;

**Reply by albertkun** [reply]
no problem since i can say that the assignment was matcha late ;)

**Reply by NellieNguyen149** [reply]
LOL ok I deserved that pun of shame o7 thank you </3

**Reply by albertkun** [reply]
There is no pun of shame in Ba Sing Se 🤐

**Comment by sachigill** [comment]
https://sachigill.github.io/24su-asiaam-191a-labs/week4/index.html
